catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -190,27 +209,37 @@ protected ResultSet getTables(Connection connection) throws SQLException {
     return metaData.getTables(databaseName, databaseName, null, JdbcConnectorUtils.TABLE_TYPES);
   }
 
-  protected ResultSet getTable(Connection connection, String tableName) throws SQLException {
+  protected ResultSet getTable(Connection connection, String databaseName, String tableName)
+      throws SQLException {
     final DatabaseMetaData metaData = connection.getMetaData();
-    String databaseName = connection.getSchema();
-    return metaData.getTables(
-        databaseName, databaseName, tableName, JdbcConnectorUtils.TABLE_TYPES);
+    return metaData.getTables(connection.getCatalog(), connection.getSchema(), tableName, null);
   }
 
-  protected ResultSet getColumns(Connection connection, String tableName) throws SQLException {
+  protected ResultSet getColumns(Connection connection, String databaseName, String tableName)
+      throws SQLException {
     final DatabaseMetaData metaData = connection.getMetaData();
-    String databaseName = connection.getSchema();
-    return metaData.getColumns(databaseName, databaseName, tableName, null);
+    return metaData.getColumns(connection.getCatalog(), connection.getSchema(), tableName, null);
   }
 
   /**
-   * @param table The result set of the table
-   * @return The properties extracted from the result set
+   * Different data sources have their own specific content that needs to be processed, such as the
```

Comment:
correct table information from the JDBC driver, because not all information could be retrieved from the JDBC driver, like the table comment in MySQL of the 5.7 version.